File: week3/data_integration.py
import pandas as pd

# Merging datasets
merged_data = pd.merge(
    left_df,
    right_df,
    how='inner',  # 'inner', 'outer', 'left', 'right'
    on='common_column'  # or left_on='col1', right_on='col2'
)

# Concatenating datasets
concatenated_data = pd.concat([df1, df2], axis=0)  # vertically
concatenated_data = pd.concat([df1, df2], axis=1)  # horizontally

File: week3/data_integration.r
library(dplyr)

# Merging datasets
merged_data <- inner_join(left_df, right_df, by = "common_column")
# Other options: left_join, right_join, full_join

# Concatenating datasets
library(tidyr)
concatenated_data <- bind_rows(df1, df2)  # vertically
concatenated_data <- bind_cols(df1, df2)  # horizontally
